client/src/pages/Login/functions.jsx

```diff
@@ -0,0 +1,25 @@
+export async function login(username, password) {
+  /*eslint no-undef: 0*/
+  console.log(username, password);
+  let promise = new Promise((res, rej) => {
+    setTimeout(() => res('An error occurred. Please try again.'), 1000);
+  });
+
+  let result = await promise;
+  return result; //return an error message string to be displayed, if an error
+}
+
+// function checks if email is valid and sends a reset password email
+export async function resetPassword(email) {
+  console.log(email);
+  let promise = new Promise((res, rej) => {
+    setTimeout(() => res(''), 1000);
```

Comment:
@Cali0707 i wasn't sure if you could return two responses... so I just displayed the message (i.e. email sent, email not found) on the front end based on the result (if it was true or false)!